queries.sql

```diff
@@ -15,3 +15,101 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT IN ('Gabumon');
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+BEGIN;
+UPDATE animals
+SET species = 'Unspecified';
+ROLLBACK;
+SELECT species FROM animals;
+COMMIT;
+
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%_mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species = 'Unspecified';
+COMMIT;
```

Comment:
Great job @prg-04 🤝

- [ ] As stated by the previous reviewer, kindly verify whether the change was made or not. You never verified the changes before and after the commit. Don't forget to update your results in the screenshots.
![transaction-1](https://github.com/prg-04/Vet_Clinic/assets/63917725/5d9ab7ec-7cda-41cb-abe0-36b1fc65795a)